extensions/podman/packages/extension/src/extension.spec.ts

```diff
@@ -3450,6 +3450,15 @@ describe('podman-mac-helper tests', () => {
   });
 });
 
+const notifySetupPodmanExpectedContent = {
```

Comment:
remark(non-blocking): personal choice, but I think that using `Mock` suffix, usually help when reading the tests to get the content, also personal thing, but I like to use uppercase with _ (just sharing, no need to update)